cms/djangoapps/contentstore/models.py

```diff
@@ -115,31 +103,107 @@ class PublishableEntityLink(models.Model):
     created = manual_date_time_field()
     updated = manual_date_time_field()
 
-    def __str__(self):
-        return f"{self.upstream_usage_key}->{self.downstream_usage_key}"
+    class Meta:
+        abstract = True
 
     @property
     def upstream_version(self) -> int | None:
```

Comment:
```suggestion
    def upstream_version_num(self) -> int | None:
```

When we say "version" in openedx-learning stuff, we almost always mean the actual version model.

Reply:
It looks like the OLX field for this [has been called just `upstream_version` already in Sumac](https://github.com/openedx/edx-platform/blob/a63a6a6fbaec6bf1c234b9b26b66c7b50c00e336/cms/lib/xblock/upstream_sync.py#L396-L404) so it's probably too late to change that, but I can change it on this `EntityLinkBase`/`ComponentLink`/`ContainerLink` model.

Reply:
Updated, but only in this Python/Django model, and not in the XBlock mixin nor REST API. 959eadd76d4733d3d14cba2fe9e1d79e473b1ac5